add_to_app/books/flutter_module_books/lib/api.dart

```diff
@@ -1,116 +1,196 @@
-// Autogenerated from Pigeon (v0.1.17), do not edit directly.
+// Autogenerated from Pigeon (v1.0.1), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
-// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import
-// @dart = 2.8
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import, unnecessary_parenthesis, prefer_null_aware_operators, omit_local_variable_types, unused_shown_name
```

Comment:
We should add a comment for each of these explaining why we're skipping these things

Reply:
filed issue for pigeon: https://github.com/flutter/flutter/issues/89671